### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by NasKhalil**
This branch includes the following features implemented:

- Add a new JavaScript files and import it as a module:
-  - it will contain methods related to the status updates (`completed`: `true` / `false`).
- Add event listener to the checkbox (`change`).
- Update items object's value for completed key upon user actions.
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/checkBoxEvent.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default (t) => {
```
Comment (SajjadAhmad14) [inline src/checkBoxEvent.js:1]: The user is unable to see items that he/she can check to mark them complete.  Please add some items in the items array and make sure they are shown on the page. Moreover, the user should be able to check them off from the list.
Your page:
list

Comment (SajjadAhmad14) [inline src/checkBoxEvent.js:1]: Please add `npm install` and `npm start` commands in README so a non-technical user can set it up locally.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Khalil, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/checkBoxEvent.js (hunk 1 above)
Comment (Emmyn5600) [reply src/checkBoxEvent.js:1]: - [x] Good Job

done

**Review by Emmyn5600 — APPROVED** [review]
STATUS: APPROVED :+1: 

Hello, team @NasKhalil  👋🏾 👋🏾 congratulations on making it so far on this milestone.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

**Pull request merged**